kitsune/sumo/form_fields.py

```diff
@@ -67,15 +67,14 @@ def to_python(self, value):
                 return []
 
         users = []
-        for username in value.split(","):
-            username = username.strip()
-            if username:
-                user = User.objects.filter(
-                    Q(username=username) | Q(profile__name=username)
-                ).first()
-                if user:
-                    if user.is_active:
-                        users.append(user)
+        usernames = [u.strip() for u in value.split(",") if u.strip()]
```

Comment:
Good idea to remove the `for` loop but isn't his already a list of usernames? Additionally why is `if u.strip()` needed?

Reply:
Lordy. That last u.strip is not needed.

Reply:
It isn't a list of usernames originally - it's whatever the value of the form field is as the user types. So the flow is get that value, convert to a clean list of strings, get a queryset of any User instance that matches on username or profile.name, validate that the user is active then return the list of User objects.

Reply:
I mean why are you validating the u.strip. This should be 
```python
usernames = [u.strip() for u in values.split() if u]
```

Reply:
I would also go for something more descriptive that `u`

Reply:
Oh - got it. Thanks!